static/index.xhtml

```diff
@@ -493,7 +493,9 @@
 				<pre id="task_logs" style="color:black;background:white;white-space:pre-wrap;"/>
 			</section>
 			<footer class="modal-card-foot">
+				<input type="hidden" id="task_id" />
 				<span class="tag is-medium" id="task_status"/>
+				<button class="button is-warning" onclick="stopTask()">Stop</button>
```

Comment:
There's an existing bug in the task logs dialog so it's not showing the task status on the bottom left (which I'll fix now). The stop button will then look out of place, so it might look better on the right, just before the close button.

Also, I think it would be nice to hide it when the task state is not running.

Reply:
Fixed